config/initializers/combine_pdf_log_patch.rb

```diff
@@ -3,18 +3,22 @@
 # Suppresses CombinePDF warning "Couldn't connect reference for" and "Form data might be lost"
 # more info: https://github.com/department-of-veterans-affairs/vets-api/pull/16705
 
-if Rails.env.test?
-  CombinePDF::PDFParser.prepend(Module.new do
-    def warn(*msgs, **)
-      msgs.reject! { |msg| msg.start_with?("Couldn't connect reference for") }
-      super
-    end
-  end)
+Rails.application.config.after_initialize do
+  require 'combine_pdf/api'
```

Comment:
this isn't working. I'm leaving a comment so this drops off of our radar as needing a review.